### ISSUE TITLE
Page Deletion after image cleanup

### ISSUE THREAD

**Issue by randommantis** [BookStackApp/BookStack#5139]
### Attempted Debugging

- [X] I have read the debugging page

### Searched GitHub Issues

- [X] I have searched GitHub for the issue.

### Describe the Scenario

I recently move our Bookstack instance from cent os 7 to ubuntu 24.04 and from bookstack beta something to [BookStack v24.05.2]. Everything moved correctly but we couldn't upload images so I ran the image clean up through the web browser but everything post upgrade was deleted. A weeks worth of page creations and revisions just deleted. Wanted to know if there was a way to restore it I unfortunately did not take a snapshot before the image cleanup.

### Exact BookStack Version

BookStack v24.05.2

### Log Content

Logs show nothing.

### Hosting Environment

PHP 8.3.6
Ubuntu 24.04
Apache/2.4.58 (Ubuntu)
mysql  Ver 8.0.37
git version 2.43.0

**Comment by ssddanbrown** [comment]
Hi @randommantis,
The image cleanup action does not touch pages or revisions at all, only image entries.

- Was anything else done around the same time?
- Can you detail how you performed the migration from the old system?
- Does anything from post-upgrade exist (users, books, activity, audit-log entries)? or does it all appear "reset" back to pre-upgrade?

**Comment by randommantis** [comment]
The issues was resolved after Regenerate References and restarting the vm (I don't know which one fixed it).

Details on the issues as it happened: 
I ran the cleanup, the webserver kept loading for 5-10 minutes and eventually turned off the vm, I restarted and noticed the pages I had made post migration were gone then I ran the cleanup till it finished correctly (after taking a snapshot this time). Regenerated References 1-2 times then rebooted the vm and came back from lunch to it seemingly working. Might've just been a weird glich/issues but I did also run "sudo chown www-data:www-data -R bootstrap/cache public/uploads storage && chmod -R 755 bootstrap/cache public/uploads storage". Anyone of these could've fixed it, probably just needed a reboot.

Everything is back now and audit log just shows I ran cleanup-images 2 times.
